mantidproject/mantid
# Reflectometry Interface (Polref): Various improvements to GUI

As stated in the issue, this PR fixes the following two issues:
- Reducing the space between widgets in the "Save ASCII" tab.
- Following the changes in #18119, resizing the tab to ensure that both "Experiment" and "Instrument" sections in the "Settings" tab can be seen (without use of scrollbar).

**To test:**
- The GUI can be opened from Interfaces->Reflectometry->ISIS Reflectometry (Polref)
- Verify that the height of the window has increased and that both sections in the "Settings" tab can be seen (and that the vertical scrollbar is no longer there).
- Verify that there is less space between widgets in the "Save ASCII" tab.



Fixes #18186.

*Does not need to be in the release notes.*

#### Reviewer ####

Please comment on the following ([full description](http://www.mantidproject.org/Individual_Ticket_Testing)):

##### Code Review #####


##### Functional Tests #####


- Does everything look good? Comment with the ship it emoji but don't merge. A member of `@mantidproject/gatekeepers` will take care of it.
